{
    "error": "HTTPSConnectionPool(host='bugzilla.clamav.net', port=443): Max retries exceeded with url: /show_bug.cgi?id=11940 (Caused by SSLError(SSLError(\"bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')])\")))"
}